Firestore/core/src/firebase/firestore/local/leveldb_key.cc

```diff
@@ -170,6 +174,12 @@ class Reader {
     return ReadLabeledString(ComponentLabel::CollectionId);
   }
 
+  /**
+   * Reads a snapshot version, encoded as a component label and a pair of seconds
+   * (int64) and nanoseconds (int32).
+   */
+  model::SnapshotVersion ReadSnapshotVersion();
```

Comment:
Note: I can't add `using model::SnapshotVersion` since the SnapshotVersion in the component labels would clash with it.